src/sidebar/components/tag-editor.js

```diff
@@ -94,9 +109,30 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
   };
 
   const handleOnInput = e => {
-    if (e.inputType === 'insertText') {
-      // Show the suggestions if the user types something into the field
-      setShowSuggestions(true);
+    if (
+      e.inputType === 'insertText' ||
+      e.inputType === 'deleteContentBackward'
+    ) {
+      if (inputEl.current.value.length === 0) {
```

Comment:
This was moved from below because I need to capture `deleteContentBackward` here now and call filter() again if there is input.